### PR TITLE
Enhanced the blog 

### LINKED ISSUE #1651: Enhance the latest blog
Add an image to the latest blog post and bold some of the bullet points for consistent formatting.
Blog - https://meshery.io/blog/simplifying-cloud-infrastructure-and-kubernetes-management

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](https://slack.meshery.io) for access.

### PR DESCRIPTION
**Pull request by Satakshijain**
**Description**
Added the images and formatted the blog

This PR fixes #1651

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: assets/images/posts/2024-02-17-simplifying-cloud-infrastructure-and-kubernetes-management/meshery-catalog.png
Comment (vishalvivekm) [inline assets/images/posts/2024-02-17-simplifying-cloud-infrastructure-and-kubernetes-management/meshery-catalog.png:1]: Let's have a light-theme friendly image:
image

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
Having images for dark as well as light theme would be great.

**Comment by sudhanshutech** [comment]
@Satakshijain done?

**Comment by Satakshijain** [comment]
Nope, I need a mockup of meshery-design for light theme

**Comment by sudhanshutech** [comment]
> Nope, I need a mockup of meshery-design for light theme

you don't need a mockup, just had to grab a light and dark mode screenshot of same and apply some existing logic which just change the images based on theme here, just that

**Pull request merged**